contrib/for-tests/porter/porter.go

```diff
@@ -33,7 +33,7 @@ const prefix = "SERVE_PORT_"
 
 func main() {
 	for _, vk := range os.Environ() {
-		parts := strings.Split(vk, "=")
+		parts := strings.SplitN(vk, "=", 2)
```

Comment:
I find it useful to have a comment or example for the use of things like `SplitN` to help others understand what is going on.